### PR TITLE
Set up component tests + Add Calendar tests

### PR DESCRIPTION
**Pull request by emilisb**
This PR introduces infra for component testing and 38 tests for `Calendar` component.

### CODE REVIEW

File: src/calendar/day/basic/index.js [hunk 1]
```diff
@@ -95,14 +85,16 @@ class Day extends Component {
       <TouchableOpacity
         testID={this.props.testID}
         style={containerStyle}
-        onPress={this.onDayPress}
+        onPress={!shouldDisableTouchEvent ? this.onDayPress : undefined}
```
Comment (emilisb) [inline src/calendar/day/basic/index.js:88]: Usually I don't like changing production code to satisfy tests, but it looked like the easiest option in this case.

Comment (Inbal-Tish) [reply src/calendar/day/basic/index.js:88]: I don't mind that. Then the same should go to 'onLongPress'

**Comment by Inbal-Tish** [comment]
Amazing work! This is a huge step forward for this repo!!!

File: src/calendar/day/basic/index.js (hunk 1 above)
Comment (emilisb) [reply src/calendar/day/basic/index.js:88]: Done

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**